utilities/rpp-performancetests/HIP_NEW/generatePerformanceLogs.py

```diff
@@ -4,8 +4,8 @@
 
 log_file_list = [
     "../OUTPUT_PERFORMANCE_LOGS_HIP_NEW/BatchPD_hip_pkd3_hip_raw_performance_log.txt",
-    "../OUTPUT_PERFORMANCE_LOGS_HIP_NEW/BatchPD_hip_pln3_hip_raw_performance_log.txt",
-    "../OUTPUT_PERFORMANCE_LOGS_HIP_NEW/BatchPD_hip_pln1_hip_raw_performance_log.txt"
+    # "../OUTPUT_PERFORMANCE_LOGS_HIP_NEW/BatchPD_hip_pln3_hip_raw_performance_log.txt",
```

Comment:
This file should ideally show as untouched in the PR. Revert these changes back to original and push.